edgexfoundry-holding/device-onvif-camera
# refactor!: Removal of manual code to add provision watchers

BREAKING CHANGE: Provision watcher folder renamed to `provisionwatchers`







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

- make test
- make docker
- /compose-builder$ make gen no-secty ds-onvif-camera (this generates docker-compose.yml with latest edgex images)
- Replace onvif docker image in docker-compose.yml with your latest image created using `make docker` from the previous step
- /compose-builder$ make up
- /bin/configure-subnets.sh without error
- /bin/map-credentials.sh without error
- Onvif devices should be registered in Edgex

## New Dependency Instructions (If applicable)
